app/CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Add a scroll view to see all the content in about screen ([#149](https://github.com/Orange-OpenSource/ods-flutter/issues/160))
```

Comment:
This is not the good issue number. `#149` should be replaced by `#160`

Reply:
oups